admiral/pkg/clusters/types.go

```diff
@@ -243,7 +290,6 @@ func HandleEventForRollout(event admiral.EventType, obj *argo.Rollout, remoteReg
 
 // helper function to handle add and delete for DeploymentHandler
 func HandleEventForDeployment(event admiral.EventType, obj *k8sAppsV1.Deployment, remoteRegistry *RemoteRegistry, clusterName string) {
-	log.Infof(LogFormat, event, "deployment", obj.Name, clusterName, "Received")
```

Comment:
Do we not need this log anymore?

Reply:
This has been pushed further up as this method is being invoked for other events.